test/unit/image_test.rb

```diff
@@ -10,4 +10,9 @@ class ImageTest < ActiveSupport::TestCase
     image.valid? #trigger validations
     assert image.errors.messages.keys.include?(:uuid)
   end
+
+  test "image scoped search for compute_resource works" do
+    resource = compute_resources(:one)
+    assert Image.search_for("compute_resource = #{resource.name}")
```

Comment:
Better might be to check the result includes one of the images assigned to this compute resource (test/fixtures/images.yml): `assert_includes Image.search_for("compute_resource = #{resource.name}"), images(:one)`